config/environments/test.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Some tests require the following languages (en, es, fr, nl, pt-BR)
+  # so we override the available languages for the test environment.
+  config.i18n.default_locale = :en
+  available_locales = [
+    "en",
+    "es",
+    "fr",
+    "nl",
+    "pt-BR"
+  ]
+  config.i18n.available_locales = available_locales
```

Comment:
Maybe we can omit the local variable and assign the value to `config.i18n.available_locales` directly :thinking:. Since there are only a few languages, we could also have everything on one line:
```
  config.i18n.available_locales = %w[en es fr nl pt-BR]
```
What do you think?